node:events:495
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::9400
    at Server.setupListenHandle [as _listen2] (node:net:1817:16)
    at listenInCluster (node:net:1865:12)
    at Server.listen (node:net:1953:7)
    at Function.listen (/Users/bero/Projects/playground-tester/node_modules/express/lib/application.js:635:24)
    at r (file:///Users/bero/Projects/playground-tester/node_modules/@wp-playground/cli/cli.js:1:805)
    at new Promise (<anonymous>)
    at j (file:///Users/bero/Projects/playground-tester/node_modules/@wp-playground/cli/cli.js:1:775)
    at Q (file:///Users/bero/Projects/playground-tester/node_modules/@wp-playground/cli/cli.js:19:185)
Emitted 'error' event on Server instance at:
    at emitErrorNT (node:net:1844:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  code: 'EADDRINUSE',
  errno: -48,
  syscall: 'listen',
  address: '::',
  port: 9400
}

Node.js v18.18.2
